cmd/thermal-recorder/config.go

```diff
@@ -1,26 +1,23 @@
 package main
 
 import (
-	"errors"
 	"io/ioutil"
-	"time"
 
+	"github.com/TheCacophonyProject/thermal-recorder/motion"
+	"github.com/TheCacophonyProject/thermal-recorder/recorder"
+	"github.com/TheCacophonyProject/thermal-recorder/throttle"
 	yaml "gopkg.in/yaml.v2"
```

Comment:
Put the yaml dep in a separate group in the middle. Imports should be organised like:

```
<stdlib imports>

<third party imports>

<local imports>
```